src/docs/services/relay/projectconfig-versioning.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: "Naming and versioning patterns in project config"
+---
+
+## Internal vs External Relay?
+
+Relay fetches project configs from an "upstream". That upstream can either be Sentry or another Relay, both have their own implementation of the same endpoint.
+
+Your configuration will flow from Sentry to either:
+
+- Relay (freight deployment = `relay`)
+
+- External Relays (customers run those, we practically have no deprecation policy for old versions)
+
+  - At some point we want to serve those requests from Relay, such that only internal Relays can hit Sentry. This would limit complexity.
```

Comment:
I think it wouldn't change much, since you still have to think about most if not all of these constraints. I'd remove this note.